frontend/lib/resolution/prims.cpp

```diff
@@ -62,14 +118,118 @@ CallResolutionResult resolvePrimCall(Context* context,
     case PRIM_IS_INSTANTIATION_ALLOW_VALUES:
     case PRIM_IS_PROPER_SUBTYPE:
     case PRIM_IS_BOUND:
+      if (ci.numActuals() == 2) {
```

Comment:
For any of these cases that are more than 5 lines or so, let's put them into their own function, to prevent `resolvePrimCall` from becoming too unwieldy.